zh_cn/README.md

```diff
@@ -0,0 +1,1627 @@
+
+# Go Concurrency Guide
+
+This guide is built on top of the some examples of the book `Go Concurrency in Go` and `Go Programming Language`
+
+[TOC]
```

Comment:
I'm think the menu it's important, even in english, because there are many terms that are not translation so easy, of course respect the original names, of patterns, like mutexs etc ...